src/js/apps/patients/sidebar/action-sidebar_app.js

```diff
@@ -9,32 +9,59 @@ import { ACTION_SHARING } from 'js/static';
 import { LayoutView } from 'js/views/patients/sidebar/action/action-sidebar_views';
 import { CommentFormView } from 'js/views/patients/shared/comments_views';
 import { ActivitiesView, TimestampsView } from 'js/views/patients/sidebar/action/action-sidebar-activity-views';
+import { AttachmentsView } from 'js/views/patients/sidebar/action/action-sidebar-attachments-views';
 
 export default App.extend({
   onBeforeStart({ action, isShowingForm }) {
     this.action = action;
+    this.hasAttachments = action.hasAttachments();
 
     this.showView(new LayoutView({
       action: this.action,
       isShowingForm,
     }));
 
-    if (!this.action.isNew()) this.getRegion('activity').startPreloader();
+    if (!this.action.isNew()) {
+      if (this.hasAttachments) this.getRegion('attachments').startPreloader();
+
+      this.getRegion('activity').startPreloader();
+    }
   },
   beforeStart() {
     if (this.action.isNew()) return;
 
     return [
       Radio.request('entities', 'fetch:actionEvents:collection', this.action.id),
       Radio.request('entities', 'fetch:comments:collection:byAction', this.action.id),
+      Radio.request('entities', 'fetch:files:collection:byAction', this.action.id),
```

Comment:
Should we only make this request if the action has file attachments? (`this.action.hasAttachments() === true`).

Or is this fine as is? Where the request is made regardless.